R/util.r

```diff
@@ -164,7 +164,15 @@ rel_path <- function(path, base = ".") {
     if (base != ".") {
       path <- file.path(base, path)
     }
-    normalizePath(path, mustWork = FALSE)
+    # normalizePath() on Windows expands to absolute paths,
```

Comment:
I wonder if we should extract out:

```R
on_windows <- function() {
  Sys.info()["sysname"] == "Windows"
}
```

And then change the signature of `rel_path()` to `rel_path <- function(path, base = ".", windows = is_windows())`. That would make testing easier.